src/Pos/Block/Logic/Slog.hs

```diff
@@ -159,14 +159,15 @@ slogApplyBlocks blunds = do
     -- Note: it's important to put blunds first
     mapM_ dbPutBlund blunds
     -- If the program is interrupted at this point (after putting on block),
-    -- we will rollback all wallet sets at the next launch.
-    onApplyBlocks blunds `catch` logWarn
+    -- we won't save any blocks.
+    callbackBlocks <- onApplyBlocks blunds
```

Comment:
1. I would call this variable `bListenerBatch` or whatever else with `Batch` suffix. When I see variable named `***Callback` I expect it to be a function (but here it is `SomeBatchOp`).
2. This `logWarn` looked strange to me, so I don't mind removing it.
3. @pva701 I think something should be done with these comments. Your comments were bad because they are about `BListener` used in wallet. But we have other `BListener`s and it doesn't make sense to have it here.